core/wallet/pem_test.go

```
@@ -26,7 +26,25 @@ MDQ1ZTVjZjU3MTQ5N2Q5ZA==
 }
 
 func tempPemFileEncrypted() string {
-	file, err := ioutil.TempFile("", "wallet*.pem")
+	file, err := os.CreateTemp("", "wallet*.pem")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+	file.WriteString(
```

golangci-lint

Error return value of `file.WriteString` is not checked

(errcheck)